### DISCUSSION TITLE
Steps to warm-up the AI-model (Screening with one single reviewer)

### DISCUSSION THREAD

**Discussion by cchiara1** [asreview/asreview#1644] [Q&A]
Hi ASReview, 

I am screening my records independently and I have a question regarding the criteria to "warm up the AI-model" (see link: https://asreview.nl/blog/seven-ways-to-integrate-asreview/#multiple). As instructed, I need to identify 5 relevant records "as validation strategy to check if the software can find these records". However, what if ASReview does not identify them or identify only some of them? There are no instructions on what to do in those circumstances. 

Thank you in advance!

**Comment by FelixWdm** [comment]
Dear @cchiara1 we assume you have one or more 'key-papers' that are part of the dataset and that represent the type of papers you would like find and will mark as relevant. To warm up the model you can manualy search for one of these papers during the project set up.

I hope this helps, best,
Felix
